apps/staking/components/ChainBanner.tsx

```diff
@@ -1,90 +1,82 @@
 'use client';
 
-import { preferredChain } from '@/lib/constants';
-import { SpecialDataTestId } from '@/testing/data-test-ids';
-import { CHAIN } from '@session/contracts/chains';
-import { Banner } from '@session/ui/components/Banner';
-import { useWallet, useWalletChain } from '@session/wallet/hooks/wallet-hooks';
-import { useTranslations } from 'next-intl';
-import { useMemo } from 'react';
-import { SwitchChainErrorType } from 'viem';
-import { toast } from '@session/ui/lib/toast';
-
+// TODO: re-implement the chain banner
 export default function ChainBanner() {
```

Comment:
You commented this out in the app layout so I don't think you need to comment out the component code here? Unless there is a specific reason? In that case a comment would be good please? 🙏

Reply:
yeah its commented out because the code is invalid, needs to be commented out because its missing import, so the TODO comment is there to come back to this later